lib/yum/diet.ex

```diff
@@ -19,7 +19,7 @@ defmodule Yum.Diet do
         Enum.map(data, fn { key, info } ->
             %Yum.Diet{
                 ref: "#{key}",
-                translation: info
+                translation: info["translation"]
```

Comment:
The `Yum.Diet` struct expects translation to be of type `Yum.Data.translation_tree` (see the type definition on line 12). But here `info["translation"]` may return `nil`, so you'll need to check for that (e.g. `info["translation"] || %{}`).